README.md

```diff
@@ -3,9 +3,9 @@
 ## Setup
 ### Expected files in a dataset directory:
 - `dataset.json`: a json file with metadata about the dataset and the names of the other files, using the keys listed below
-- featureDefs: a json describing the measured features in this dataset, *the order of this document has to match the order of the values in the `featuresData` file. Also if no displayOrder is given it will be used as the display order on the website. *
-- featuresData: a json listing the per cell data
-- cellLineData: a json of the cell line definitions
+- featureDefsPath: path to a json describing the measured features in this dataset, *the order of this document has to match the order of the values in the `featuresData` file. Also if no displayOrder is given it will be used as the display order on the website. *
+- featuresDataPath: path to a json listing the per cell data
+- cellLineDataPath: path to a json of the cell line definitions
```

Comment:
I think in this section, these lines are supposed to be describing the contents of files themselves, not the paths